docs/src/architecture/accounts.md

```diff
@@ -64,6 +85,7 @@ The process is as follows:
 * Alice consumes Bob's note to receive the asset in a transaction
 * Depending on the account storage mode (private vs. public) and transaction type (local vs. network) the Operator receives the new Account ID eventually and - if the transaction is correct - adds the ID to the Account DB
 
+A user can simply use the Miden Client or call [this](https://github.com/0xPolygonMiden/miden-base/blob/main/miden-lib/src/accounts/wallets/mod.rs) from his application to create a new account, see [here](TODO LINK TO CLIENT ONCE READY).
```

Comment:
I would change this sentence to: 

For a user to create an account we have 2 solutions at the moment: 
- Use the [Miden Client](https://github.com/0xPolygonMiden/miden-client/tree/main) as a wallet
- Use the [Miden Base](https://github.com/0xPolygonMiden/miden-base) builtin functions for wallet creation: [Basic wallet](https://github.com/0xPolygonMiden/miden-base/blob/4e6909bbaf65e77d7fa0333e4664be81a2f65eda/miden-lib/src/accounts/wallets/mod.rs#L29), [Fungible faucet](https://github.com/0xPolygonMiden/miden-base/blob/4e6909bbaf65e77d7fa0333e4664be81a2f65eda/miden-lib/src/accounts/faucets/mod.rs#L29)

Reply:
I am curious about why we have this naming convention throughout Miden of "Basic", is it needed? 

What would a "not basic" wallet account be? 

What would a "not basic" fungible faucet account be?

Reply:
Excellent question. I expect that in the future we will have way more advanced wallets. A bit like Gnosis Safe, but in cooler, like https://github.com/0xPolygonMiden/examples/discussions/140